.goreleaser.yaml

```diff
@@ -3,42 +3,127 @@ before:
     - wget https://github.com/CosmWasm/wasmvm/releases/download/{{ .Env.LIBWASM_VERSION }}/libwasmvm_muslc.x86_64.a -O /lib/libwasmvm_muslc.a
     - wget https://github.com/CosmWasm/wasmvm/releases/download/{{ .Env.LIBWASM_VERSION }}/libwasmvm_muslc.aarch64.a -O /usr/local/lib/libwasmvm_muslc.a
     - apt install musl-dev=1.2.2-1
+env:
+  - CGO_ENABLED=1
 builds:
-  - id: linux-amd64
+  - id: archwayd-linux-amd64
     main: ./cmd/archwayd
     binary: archwayd
     goos:
       - linux
     goarch:
       - amd64
     env:
-      - CGO_ENABLED=1
       - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
     flags:
       - -mod=readonly
+      - -trimpath
     ldflags:
       - -s -w -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X github.com/cosmos/cosmos-sdk/version.Name=archway -X github.com/cosmos/cosmos-sdk/version.AppName=archway -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }}
       - -linkmode=external
-      - -extldflags '-Wl,-z,muldefs -static -lm'
+      - -extldflags '-Wl,-z,muldefs --static -lm'
     tags:
       - netgo
       - muslc
-  - id: linux-arm64
+  - id: archwayd-linux-arm64
     main: ./cmd/archwayd
     binary: archwayd
     goos:
       - linux
     goarch:
       - arm64
     env:
-      - CGO_ENABLED=1
```

Comment:
why remove this?

Reply:
because it's added in the top for all the build phases